toxicity/test/test_linear_predictor.py

```diff
@@ -0,0 +1,52 @@
+import numbers
+import unittest
+import pathmagic  # noqa
+from linear_predictor import LogisticPredictor
+import utils
+import pandas as pd
+from preprocessing import tf_idf
+
+train_file = "../data/train.csv"
+test_file = "../data/test.csv"
+
+
+class TestLinearPredictor(unittest.TestCase):
+    # We will use a chunk of the dataset of n rows
+    number_of_rows = 1000
```

Comment:
I dont think we need this here, just do `self.train = pd.read_csv(train_file, nrows=1000)` in the SetUp

Reply:
I will leave in order to be things clear.